paddlets/analysis/analysis_report.py

```diff
@@ -139,7 +139,7 @@ def export_docx_report(self, path: str = ".", file_name: str = "analysis_report.
         document.save(path)
         logger.info(f"save report succcess, save at {path}")
     
-    def export_json_report(self, log: bool = True) -> Dict:
+    def export_json_report(self, log: Optional[bool] = True) -> Dict:
```

Comment:
should NOT be None, refers to https://docs.python.org/3/library/typing.html#typing.Optional